.test-infra/OWNERS

```diff
@@ -0,0 +1,4 @@
+# See the OWNERS docs at https://s.apache.org/beam-owners
```

Comment:
This URL doesn't resolve for me: https://s.apache.org/beam-owners

Reply:
This was supposed to link to https://beam.apache.org/contribute/#reviews .
I forgot to ask someone with permissions to set that. I'd appreciate it if you could set it.

Reply:
Done.